Simple-PHP-IPAM/upgrade.sh

```
@@ -237,10 +225,77 @@ if [[ -n "$CHOWN_BIN" ]]; then
+          --from="$current_driver" --from-dsn="$src_dsn" \
+          --to="$target_driver" --to-dsn="$target_dsn" \
+          --to-user="$target_user" --to-pass="$target_pass" \
+          --force && migrate_ok=1
+
+        if [[ "$migrate_ok" -eq 1 ]]; then
+          export IPAM_NEW_DRIVER="$target_driver"
+          export IPAM_NEW_DSN="$target_dsn"
+          export IPAM_NEW_USER="$target_user"
+          export IPAM_NEW_PASS="$target_pass"
+          "$PHP_BIN" -r '
+            $path = getenv("TARGET_DIR") ?: "'$TARGET_DIR'/config.php";
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)